Extension/src/LanguageServer/extension.ts

```diff
@@ -743,6 +748,27 @@ async function onCreateDeclarationOrDefinition(args?: any): Promise<void> {
     return getActiveClient().handleCreateDeclarationOrDefinition(false, args?.range);
 }
 
+async function onExtractToFunction(extractAsGlobal: boolean, extractAsMemberFunction: boolean, sender?: string): Promise<void> {
```

Comment:
extractAsGlobal, extractAsMemberFunction, ExtractToFunction can be enums instead of checking a bool.

Reply:
Using 2 bools seems simpler. What enum did you have in mind?

Reply:
The enum names would simply reflect the type of extraction being done. Having an enum would avoid errors of passing in incorrect bool values. So instead of a combination of true or false, you can pass in an enum extract type and would be readable what the caller is intending to do based on the enum value passed in.

Reply:
Just a coding suggestion though. It's not necessary.